### PR TITLE
Documenting System Information Resource

### PR DESCRIPTION
**Pull request by dino-saurabh**
1. Added response and missing examples

### CODE REVIEW

**Comment by dino-saurabh** [comment]
@Akayeshmantha  please review !!

File: source/includes/SystemInformation/system_information.md [hunk 1]
```diff
@@ -22,7 +22,96 @@
...
+            "Reporting UI Module": "1.6.0 ",
+            "Appointment Scheduling Module": "1.12.0 ",
+            "Rest Web Services OMOD": "2.28.0.df1459 ",
+            "Legacy UI Module": "1.6.0 ",
+            "Data Exchange Module": "1.3.4 ",
+            "Reference Demo Data Module": "1.4.5 ",
+            "UI Library Module": "2.0.6 "
+        }
+    }
+}
```
Comment (dino-saurabh) [inline source/includes/SystemInformation/system_information.md:115]: @Akayeshmantha  I would like your view on how should I depict this response it seems quite large ?

Comment (Akayeshmantha) [reply source/includes/SystemInformation/system_information.md:115]: that's fine let's give it as it is

**Review by Akayeshmantha — APPROVED** [review]
LGTM

**Pull request merged**